src/Components/SingleReview.jsx

```diff
@@ -32,8 +41,22 @@ const Review = () => {
         alt={singleReview.Review_name}
       />
       <p>{singleReview.review_body}</p>
-      <p>{singleReview.created_at}</p> <p>votes: {singleReview.votes}</p>
-      <Link to="/">Home</Link> <Link to="/reviews">All Reviews</Link>;
+      <p>Votes: {singleReview.votes}</p>
+      <p>{singleReview.created_at}</p>
+      <Link to="/">Home</Link> <Link to="/reviews">All Reviews</Link>
+      <h2>Comments</h2>
+      <ul>
+        {comments.map((comment) => {
+          return (
```

Comment:
Although it's not mandatory, it makes sense to extract each comment into a comment card with props. so you could return: `<CommentCard review_id={comment.review_id} body ={review.body} />` etc.